app/Helpers/GeometryHelper.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Helpers;
+
+use App\Models\V2\PolygonGeometry;
+use App\Models\V2\Projects\Project;
+use Illuminate\Support\Facades\Log;
+
+class GeometryHelper
+{
+    public function centroidOfProject($projectUuid)
+    {
+        $project = Project::where('uuid', $projectUuid)->first();
+
+        if (! $project) {
+            return null;
+        }
+
+        $sitePolygons = $project->sitePolygons;
+
+        if ($sitePolygons->isEmpty()) {
+            return null; // Return null if no polygons are found for the given projectUuid
+        }
+
+        $polyIds = $sitePolygons->pluck('poly_id')->toArray();
```

Comment:
This will be more efficient in the DB this way:
```
$polyIds = $project->sitePolygons()->pluck('poly_id')->toArray();
if ($polyIds->isEmpty()) {
  return null; // Return null if no polygons are found for the given projectUuid
}
```

Keep in mind that when accessing an association, if you access it without the function call (`->sitePolygons` vs `->sitePolygons()`), it will aggressively load all columns for every record. So if you only need a single column, accessing it this way is more efficient.